lib/transforms/subsetGoogleFonts.js

```diff
@@ -373,6 +382,24 @@ module.exports = function (options) {
                         fontStyleBundle = new AssetGraph.Css(assetGraph.resolveAssetConfig({
                             text: source
                         }));
+
+                        if (fontDisplay) {
+                            fontStyleBundle.parseTree.nodes.forEach(function (atRule) {
+                                var declarations = atRule.nodes;
```

Comment:
Hmm, this won't detect `@font-face` rules nested inside `@media` queries.

Also, we should only dive into `@font-face` rules here in the code that follows? (`atRule.name === 'font-face'`)

Reply:
Yeah, this is pretty native and will probably only work for Google fonts, where I can make a lot of assumptions about the stylesheet. I think the generic case might be a bit harder

Reply:
Fair :)